package.json

```diff
@@ -48,16 +48,18 @@
     "eslint-test-generator": "1.0.5",
     "expect.js": "0.3.1",
     "fb-watchman": "1.9.0",
+    "fetch-mock": "^5.8.1",
```

Comment:
All other packages have a specific version number. I'm guessing this was done via `npm install --save-dev <package>` which uses `^`.

@minasmart should we stick to strict versions?